### PR TITLE
Added Readme Badges

### PR DESCRIPTION
**Pull request by UtkarshBhatthere**

### CODE REVIEW

File: microceph/database/config.mapper.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func GetConfigItems(ctx context.Context, tx *sql.Tx, filters ...ConfigItemFilter
 	var err error
 
 	// Result slice.
-	objects := make([]ConfigItem, 0)
+	var objects []ConfigItem
```
Comment (sabaini) [inline microceph/database/config.mapper.go:113]: The *.mapper.go files are generated by lxd-generate, we should avoid editing them if at all possible. I would suggest to report this issue upstream with the lxd project

File: microceph/database/disk.mapper.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func GetDisks(ctx context.Context, tx *sql.Tx, filters ...DiskFilter) ([]Disk, e
 	var err error
 
 	// Result slice.
-	objects := make([]Disk, 0)
+	var objects []Disk
```
Comment (sabaini) [inline microceph/database/disk.mapper.go:128]: Same as above -- generated code

File: microceph/database/service.mapper.go [hunk 3]
```diff
@@ -133,7 +133,7 @@ func GetServices(ctx context.Context, tx *sql.Tx, filters ...ServiceFilter) ([]S
 	var err error
 
 	// Result slice.
-	objects := make([]Service, 0)
+	var objects []Service
```
Comment (sabaini) [inline microceph/database/service.mapper.go:136]: Same as above -- generated code

**Review by sabaini — CHANGES_REQUESTED** [review]
Hi @UtkarshBhatthere the badges lgtm but if possible we should avoid updating the generated mapper.go files

File: microceph/database/config.mapper.go (hunk 1 above)
Comment (UtkarshBhatthere) [reply microceph/database/config.mapper.go:113]: Hmm, that's fair... reversing the changes.

**Comment by sabaini** [comment]
lgtm!

**Pull request merged**